### PR TITLE
Sb improve logging save view error

### PR DESCRIPTION
**Pull request by SteffenBrinckmann**

### CODE REVIEW

File: pasta_eln/database.py [hunk 1]
```diff
@@ -567,7 +567,9 @@ def saveView(self, designName:str, viewCode:dict[str,str]) -> None:
     try:
       designDoc.save()
     except Exception:
-      print('**ERROR dsv01: something unexpected has happend. Log-file has traceback')
+      print('**ERROR dsv01: something unexpected has happend.')
```
Comment (jmurugan-fzj) [inline pasta_eln/database.py:570]: Type in the error message, "happened"..please correct and also so we only log the exception right?

File: pasta_eln/GUI/form.py [hunk 2]
```diff
@@ -48,8 +48,10 @@ def __init__(self, comm:Communicate, doc:dict[str,Any]):
                       if hasattr(self.comm.backend, 'configuration') else 300
       imageW, self.imageL = widgetAndLayout('V', mainL)
       Image(self.doc['image'], self.imageL, anyDimension=width)
-      imageW.setContextMenuPolicy(Qt.CustomContextMenu)
-      imageW.customContextMenuRequested.connect(lambda pos: initContextMenu(self, pos))
+      if '_id' in self.doc:
+        self.docID= doc['_id']  #required for hide to work
```
Comment (jmurugan-fzj) [inline pasta_eln/GUI/form.py:52]: Is this part of some other issue?

File: pasta_eln/database.py (hunk 1 above)
Comment (SteffenBrinckmann) [reply pasta_eln/database.py:570]: I corrected the spelling. The exception is written into the log for easy sending to developers, inspection,... The terminal gives just a error message to make the user more aware. Yes, a pop-up on the screen would be even better. Future issue.

File: pasta_eln/GUI/form.py (hunk 2 above)
Comment (SteffenBrinckmann) [reply pasta_eln/GUI/form.py:52]: Don't remember; but you can recreate the issue: open any measurement and the editor, and PASTA would write "widget has no docID" not causing any other big problems. This ensures that widget 'self' has a docID.

**Review by jmurugan-fzj — APPROVED** [review]

**Pull request merged**